mantidproject/mantid
# Enhance ExportTimeSeriesProperty and AddSampleLog for advanced event filtering

Enhance the functionality of ExportTimeSeriesProperty and AddSampleLog such that any sample log can be exported from a workspace, be processed and added back to the workspace.

**To test:**

Load an arbitrary workspace, export a sample log either in integer or double TimeSeriesProperty to a MatrixWorkspace by ExportTimeSeriesProperty().  You can choose the time be epoch time or relative time to run start.  Then add it back by AddSampleLog.  Check whether the original one and newly added one are identical.

Also, you can try to export a sample log with its first derivative. 



Fixes #19324.

Release note is added to docs/source/release/v3.10.0/framework.rst 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
